demo/midgard-sdk/src/tx-builder/user-events/deposit/index.ts

```diff
@@ -1 +1,65 @@
+import { Data, Integer, LucidEvolution, Script, TxBuilder} from "@lucid-evolution/lucid";
+import { DepositDatum, DepositInfo, MintRedeemer } from "./types.js";
+import { POSIXTime } from "@/tx-builder/common.js";
+import { sha3_256 } from "@noble/hashes/sha3";
```

Comment:
We are using [`blake2b256`](https://github.com/Anastasia-Labs/midgard/blob/f37f06bb3d48340e9852ac128d33e800d3e09f18/onchain/aiken/lib/midgard/user-events.ak#L54-L56).

We also have a helper you might be able to use:
```suggestion
import { hashHexWithBlake2b256 } from "@/utils/common.js";
```